keps/sig-node/2570-memory-qos/README.md

```diff
@@ -619,6 +634,8 @@ checking if there are objects with field X set) may be a last resort. Avoid
 logs or events for this purpose.
 -->
 
+An operator could run ls `/sys/fs/cgroup/kubepods.slice/kubepods-burstable.slice/kubepods-burstable-pod<SOME_ID>.slice` on a node with cgroupv2 enabled to confirm the presence of `memory.min` file which tells us that the feature is in use by the workloads.
+
```

Comment:
I think if the FG is set, and the users set any requests/limits, this feature will be in use, right? It doesn't require special opt in by the user in their PodSpec?

If so, please document that. Again I think we need the metric on breaches.

Reply:
Done.